sdk/resources/mgmt/src/main/java/com/azure/management/UserAgentPolicy.java

```diff
@@ -63,7 +63,7 @@ public UserAgentPolicy(HttpLogOptions httpLogOptions, Configuration configuratio
         }
 
         if (configuration == null) {
-            this.configuration = Configuration.getGlobalConfiguration();
+            this.configuration = Configuration.getGlobalConfiguration().clone();
```

Comment:
Okay, make sense.

Reply:
This might be a question whether we support the users to change configuration when running the client. It's not allowed by credential from Azure Identity. Maybe we should consider to align with the same rule.

Reply:
I think configuration should be decided by user no matter for any purpose.